source/Halibut.Tests/UsageFixture.cs

```diff
@@ -246,18 +246,26 @@ public async Task OctopusCanSendAndReceiveComplexObjects_WithMultipleComplexChil
 
                 response.Child1.Should().NotBeSameAs(request.Child1);
                 response.Child1!.ChildPayload1.Should().NotBeSameAs(request.Child1.ChildPayload1);
-                response.Child1.ChildPayload1!.ReadAsString().Should().Be(childPayload1);
+                (await response.Child1.ChildPayload1!.ReadAsString(CancellationToken)).Should().Be(childPayload1);
                 response.Child1.ChildPayload2.Should().NotBeSameAs(request.Child1.ChildPayload2);
-                response.Child1.ChildPayload2!.ReadAsString().Should().Be(childPayload2);
+                (await response.Child1.ChildPayload2!.ReadAsString(CancellationToken)).Should().Be(childPayload2);
                 response.Child1.ListOfStreams.Should().NotBeSameAs(request.Child1.ListOfStreams);
-                response.Child1.ListOfStreams!.Select(x => x.ReadAsString()).ToList().Should().BeEquivalentTo(list);
+                (await response.Child1.ListOfStreams!.ToAsyncEnumerable()
```

Comment:
Nit: It would be nice if we didn't have to do the `ToAsyncEnumerable` -> `ToListAsync` dance.

Is there a method that does that out of the box? 🤔 The only way I can think of is to make our own extension method.

But that's just to make some syntactic sugar. Just a thought 🙂